components/nftMarket/AllBidsView.tsx

```diff
@@ -0,0 +1,227 @@
+// import { useTranslation } from 'next-i18next'
+import { useEffect, useState } from 'react'
+import {
+  useAuctionHouse,
+  useBids,
+  useListings,
+  useLoadBids,
+} from 'hooks/market/useAuctionHouse'
+import { toUiDecimals } from '@blockworks-foundation/mango-v4'
+import { MANGO_MINT_DECIMALS } from 'utils/governance/constants'
+import { useWallet } from '@solana/wallet-adapter-react'
+import metaplexStore from '@store/metaplexStore'
+import { Bid, Listing, PublicBid, PublicKey } from '@metaplex-foundation/js'
+import BidNftModal from './BidNftModal'
+import mangoStore from '@store/mangoStore'
+import {
+  Table,
+  TableDateDisplay,
+  Td,
+  Th,
+  TrBody,
+  TrHead,
+} from '@components/shared/TableElements'
+import { ImgWithLoader } from '@components/ImgWithLoader'
+import NftMarketButton from './NftMarketButton'
+import { abbreviateAddress } from 'utils/formatting'
+import { NoSymbolIcon } from '@heroicons/react/20/solid'
+
+const AllBidsView = () => {
+  const wallet = useWallet()
```

Comment:
Can you destructure the variables you're using from `useWallet`? It's a bit more concise and prevents having to call `wallet.wallet` when you need to access the actual wallet object.